sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/UnsupportedOperationsSuite.scala

```diff
@@ -413,36 +414,103 @@ class UnsupportedOperationsSuite extends SparkFunSuite {
     batchStreamSupported = false,
     streamBatchSupported = false)
 
-  // Left outer joins: *-stream not allowed
+  // Left outer joins: *-stream not allowed with default condition
```

Comment:
The condition that testBinaryOperationInStreamingPlan uses if left unspecified. I've removed the reference since it wasn't there originally, but let me know if you think some additional comment belongs here.